lib/apicall.js

```diff
@@ -0,0 +1,9 @@
+export async function apiRandomPageCall() {
```

Comment:
Usually, naming conventions for functions that retrieve data have "fetch" or "get" in them. Also, if you're sticking with the file name it should be plural :)